sentry-android-ndk/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+
+plugins {
+    id("com.android.library")
+}
+
+android {
+    compileSdkVersion(Config.Android.compileSdkVersion)
+    buildToolsVersion(Config.Android.buildToolsVersion)
+
+    defaultConfig {
+        targetSdkVersion(Config.Android.targetSdkVersion)
+        javaCompileOptions {
+            annotationProcessorOptions {
+                includeCompileClasspath = true
+            }
+        }
+
+        minSdkVersion(21)
+        externalNativeBuild {
+            cmake {
+                arguments.add(0, "-DANDROID_STL=c++_static")
+                arguments.add(0, "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON")
+            }
+        }
+        ndk {
+            abiFilters("x86", "armeabi-v7a", "x86_64", "arm64-v8a")
```

Comment:
if we wanna support non-NEON devices, we might need to look into it.
https://android.googlesource.com/platform/ndk/+/master/docs/BuildSystemMaintainers.md#NEON

Reply:
https://android-developers.googleblog.com/2019/10/introducing-ndk-r21-our-first-long-term.html NDK 21 has NEON enabled by default, so I'd say it's gonna be fixed by itself when we bump up.